### PR TITLE
added ISV how to for checking SCIM attribute mappings

### PR DESCRIPTION
**Pull request by yuvalalter-okta**
## Description:
- Converted wiki page(https://oktawiki.atlassian.net/wiki/spaces/eng/pages/511575437/Check+the+Attributes+for+Your+Application+and+their+corresponding+Mappings) to SCIM how-to doc for ISV's

### Resolves:

* [OKTA-183504](https://oktainc.atlassian.net/browse/OKTA-183504)

### CODE REVIEW

**Comment by jakubvul** [comment]
@yuvalalter-okta I was talking to @nbarbettini about this and he was wondering when you were expecting for this to go live?

**Comment by yuvalalter-okta** [comment]
@jakubvul I was hoping to have this go live end of this week or sometime next week. Is this interfering with any other timelines?

File: _source/_standards/SCIM/check-app-attributes.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
+---
+layout: docs_page
+title: Check the Attributes for Your Application and their corresponding Mappings
+excerpt: Check the attributes and the corresponding mappings for a SCIM app that has been created through the app wizard.
+---
+
+# Check the Attributes For Your Application and Their Corresponding Mappings #
```
Comment (motikagan-okta) [inline _source/_standards/SCIM/check-app-attributes.md]: The hash character(s) for headings are only needed at the beginning of the line, not the end.

Comment (yuvalalter-okta) [reply _source/_standards/SCIM/check-app-attributes.md]: I added them to the end for the sake of readability, should I only be using it at the beginning of lines?

File: _source/_standards/SCIM/check-app-attributes.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+8.  On the pop-up that appears, there will be two drop-down fields. On
+    the first drop-down, select **Map from SCIM 2.0 Test App Profile**.
+    On the second drop-down, choose the Okta profile attribute you would
+    like to map the SCIM attribute to. Click **Save**.
+    {% img scim_attributes_screenshots/image14.png alt:"Map from SCIM 2.0 Test App Profile" width:"625px" %}
+
+9.  Repeat the same step for all other SCIM attributes that you would
+    like to modify the mapping for (App to Okta).
+
+10. Done!
```
Comment (motikagan-okta) [inline _source/_standards/SCIM/check-app-attributes.md]: I would delete this step.

**Comment by jakubvul** [comment]
@yuvalalter-okta our ISV content is now hosted on [okta.com/integrate](https://www.okta.com/integrate/). Ping me and we can figure out how to proceed.

**Pull request closed without merging**